src/main/java/com/redhat/devtools/lsp4ij/client/SettingsHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.redhat.devtools.lsp4ij.client;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+public class SettingsHelper {
+
+    private SettingsHelper() {
+    }
+
+    public static JsonElement findSettings(String[] sections, JsonObject jsonObject) {
+        JsonElement current = jsonObject;
+        for (String section : sections) {
+            if (current instanceof JsonObject object) {
```

Comment:
Rename object with child

Reply:
I agree that "object" was a terrible name. Child would not be a correct name though because the actual child is retrieved with get() one line below. This variable is just "current" casted to JsonObject. I therefore called it currentObject